x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/inference/trainedmodel/ensemble/Ensemble.java

```diff
@@ -250,6 +250,10 @@ public int hashCode() {
 
     @Override
     public void validate() {
+        if (this.models.isEmpty()) {
+            throw ExceptionsHelper.badRequestException("[trained_models] must not be empty");
```

Comment:
Would it make sense to use `TRAINED_MODELS.getPreferredName()` here rather than literal?